spec/main_test_spec.rb

```diff
@@ -0,0 +1,44 @@
+# main tests
+require './bin/main.rb'
+RSpec.describe Enumerable do
+  array = [1, 2, 3, 4, 5]
```

Comment:
Use `let` instead of local variable, how? [check it here](http://www.betterspecs.org/#let)